go.mod

```diff
@@ -4,8 +4,9 @@ go 1.16
 
 require (
 	github.com/gin-gonic/gin v1.6.3
+	github.com/go-chi/chi/v5 v5.0.1
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/viper v1.7.1
-	github.com/stretchr/testify v1.7.0
```

Comment:
this is an awkward change, that indicates the line can be removed because we dropped all _test.go files? 

can we at least have one unit test back?

Reply:
yes thx Dirk.. I have remove the test but I plan to add some unit test later on. thx!

Reply:
Please keep one unit test :) really isn't great practive to remove it first ;)

Reply:
https://github.com/SUSE/console-for-sap-applications/pull/7/commits/67a589510ef9822a20a6bfdf6361772e7110b46c done in that